denizen_scripts/survival/repo-link/MobScripts/SlimeStuff.dsc

```diff
@@ -0,0 +1,45 @@
+slime_mob_handler:
+  type: world
+  debug: false
+  events:
+    on player damaged by slime:
+      - if <context.damager.is_mythicmob> && <context.damager.MythicMob.level.is[MORE].to[5]>:
+        - determine passively cancelled
```

Comment:
we don't want to determine this cancelled if we're determining damage down further below
```suggestion
```